### ISSUE TITLE
VTKvolume example in the Gallery is not doing anything

### ISSUE THREAD

**Issue by thoth291** [holoviz/panel#930]
Here is what I do: https://panel.pyviz.org/reference/panes/VTKVolume.html#gallery-vtkvolume
And I get this instead (using Chrome on my Mac).
<img width="808" alt="Screen Shot 2020-01-09 at 1 29 12 AM" src="https://user-images.githubusercontent.com/2971670/72047151-7c8be600-327f-11ea-9452-4fa71e75947a.png">
I'm using the version from the master branch.
Using the latest available does **not** fixing it for me.

**Comment by ceball** [comment]
I think there's probably not enough information in your request for people to help you. Please could you say how you installed panel, where you are using panel ("classic" jupyter notebook?), what other packages you have installed (e.g. conda list or the equivalent for your setup), whether there are any errors in chrome's developer tools console, and so on?

Might also be worth checking that basic things work for you (e.g. `pn.pane.Markdown("""#h1""")`).

**Comment by philippjfr** [comment]
As @ceball indicated your issue is lacking a lot of information, e.g. how did you install Panel from master? What version of bokeh do you have?

**Comment by thoth291** [comment]
I tried yesterday night few other ways to install and none of them worked.
Now I simply tried from scratch again using this list of commands and it worked.
```bash
conda install -c pyviz panel python=3 nodejs jupyter pyvista holoviews ipywidgets
```
Surprisingly installing the same way into my original environment also worked.
I'm puzzled... Not the first time... Not the last time...
Sorry for inconvenience.
And thank you, @ceball  and @philippjfr for your reply and help!

Needless to say - your project is one of the best in the Python world!
